### DISCUSSION TITLE
水贴

### DISCUSSION THREAD

**Discussion by 123Windows31** [daizihan233/HanBot#24] [General]
我是傻逼

**Comment by daizihan233** [comment]
~~《 认 清 自 我 》~~

**Comment by daizihan233** [comment]
需要我给你设个**置顶**或者**锁定**_**（即没有人可以回复/删除此贴）**_ 吗[doge]

**Comment by 123Windows31** [comment]
需要（）




------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "daizihan233/HanBot"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2022年3月21日(星期一) 晚上9:22
***@***.***&gt;;
抄送:&nbsp;" ***@***.******@***.***&gt;;
主题:&nbsp;Re: [daizihan233/HanBot] 水贴 (Discussion #24)





 
需要我给你设个置顶或者锁定（即没有人可以回复/删除此贴） 吗[doge]

**Comment by daizihan233** [comment]
~~那您需要锁定还是置顶还是都要呢（）~~

**Comment by 123Windows31** [comment]
<del>都要（）</del>

**Comment by daizihan233** [comment]
好了（）

**Comment by daizihan233** [comment]
算了不置顶了，置顶有点丑（）

**Comment by daizihan233** [comment]
我又给你解锁力（雾）
